WooCommerce/WooCommerceTests/ViewRelated/Products/BottomSheetListSelector/ProductsSortOrderBottomSheetListSelectorCommandTests.swift

```diff
@@ -25,8 +34,12 @@ final class ProductsSortOrderBottomSheetListSelectorCommandTests: XCTestCase {
     func testIsSelectedValueAfterChange() {
```

Comment:
It looks like `testIsSelectedValueAfterChange` was originally testing the that `command.isSelected` was returns the correct boolean value for the sort value passed to `command.handleSelectedChange()`. However, `handleSelectedChange` doesn't do anything that affects that since it now just calls a callback function.

We can repurpose this to only test the initial value. Something like this: 

```swift
    func test_isSelected_returns_true_if_given_the_initial_value() {
        // Arrange
        let command = ProductsSortOrderBottomSheetListSelectorCommand(selected: .nameAscending) { (selected) in
            // noop
        }

        // Action
        let isSelected = command.isSelected(model: .nameAscending)

        // Assert
        XCTAssertTrue(isSelected)
    }
```

We can also add another unit test that returns `false` if the incorrect sort value is passed but I leave that up to you if you want to do that. It's not that critical.